src/CardDeck.php

```diff
@@ -59,7 +59,7 @@ class CardDeck extends View
     public $search = [VueComponent\ItemSearch::class];
 
     /** @var array Default notifier to perform when model action is successful * */
-    public $notifyDefault = [JsToast::class, 'settings' => ['displayTime' => 5000]];
+    public $notifyDefault = [JsToast::class, 'settings' => ['class' => 'success', 'displayTime' => 5000]];
```

Comment:
This change will introduce inconsistency in the code.

`class.*success` regex shows this is not how CRUD handles it

JsToast default is set in https://github.com/atk4/ui/blob/3c592b8f10fe67c61f179c5c8723b07f8ab754b9/src/Js/JsToast.php#L37 and it defaults to `success`.

Please adjust the PR.

Reply:
Please re-check PR.